tests/phpunit/tests/admin/wpListTable.php

```diff
@@ -104,4 +122,226 @@ public function data_should_only_add_primary_column_when_needed() {
 
 		return $datasets;
 	}
+
+	/**
```

Comment:
This alignment is fixed in a commit which I'll push along with commits to fix the failing tests.

Reply:
Fixed in [92a05b4](https://github.com/WordPress/wordpress-develop/pull/2828/commits/92a05b4a83aeb81644e7b2e0b6eb29a709e76561).